src/test/scala/com/typesafe/npm/NpmSpec.scala

```diff
@@ -15,7 +15,7 @@ class NpmSpec extends Specification with NoTimeConversions {
   def withEngine[T](block: ActorRef => T): T = {
     val system = ActorSystem("test-system", classLoader = Some(this.getClass.getClassLoader))
     try {
-      val engine = system.actorOf(Trireme.props(), "engine")
+      val engine = system.actorOf(Node.props(), "engine")
```

Comment:
Actually the `npm-tester` projects fails for me when having `npm` 4.2.0 and node `v7.9.0` installed on my system (Ubuntu 16.04). Is it working for you (after making my requested change)?